src/link/MachO.zig

```diff
@@ -217,6 +217,7 @@ pub fn createEmpty(
         .undefined_treatment = if (allow_shlib_undefined) .dynamic_lookup else .@"error",
         .lib_dirs = options.lib_dirs,
         .framework_dirs = options.framework_dirs,
+        .force_load_objc = options.force_load_objc,
```

Comment:
@andrewrk this is the meaningful change. The machinery to implement `-ObjC` was done by me when upstreaming the revamped linker a few weeks back already - however I didn't plug in parsing of `-ObjC` into the CLI.

Reply:
ahhh! you already did all the work and simply didn't hook it up yet, I see :)

Reply:
Correct!